seaseq-control.wdl

```diff
@@ -462,45 +466,48 @@ workflow seaseq {
         call fastqc.fastqc as c_mergebamfqc {
             input:
 	        inputfile=c_mergebam.mergebam,
-                default_location='CONTROL/' + sub(basename(c_mergebam.mergebam),'\.sorted\.b.*$','') + '/QC/FastQC'
+                default_location='CONTROL/' + sub(basename(c_mergebam.mergebam),'.sorted.b.*$','') + '/QC/FastQC'
         }
 
         call samtools.indexstats as c_mergeindexstats {
             input:
                 bamfile=c_mergebam.mergebam,
-                default_location='CONTROL/' + sub(basename(c_mergebam.mergebam),'\.sorted\.b.*$','') + '/BAM_files'
+                default_location='CONTROL/' + sub(basename(c_mergebam.mergebam),'.sorted.b.*$','') + '/BAM_files'
         }
 
         if ( defined(blacklist) ) {
             # remove blacklist regions
             String c_string_blacklist = "" #buffer to allow for blacklist optionality
-            File c_blacklist_ = select_first([blacklist, c_string_blacklist])
+            File c_blacklist_m = select_first([blacklist, c_string_blacklist])
             call bedtools.intersect as c_merge_rmblklist {
                 input :
                     fileA=c_mergebam.mergebam,
-                    fileB=c_blacklist_,
-                    default_location='CONTROL/' + sub(basename(c_mergebam.mergebam),'\.sorted\.b.*$','') + '/BAM_files',
+                    fileB=c_blacklist_m,
+                    default_location='CONTROL/' + sub(basename(c_mergebam.mergebam),'.sorted.b.*$','') + '/BAM_files',
                     nooverlap=true
             }
             call samtools.indexstats as c_merge_bklist {
                 input :
                     bamfile=c_merge_rmblklist.intersect_out,
-                    default_location='CONTROL/' + sub(basename(c_mergebam.mergebam),'\.sorted\.b.*$','') + '/BAM_files'
+                    default_location='CONTROL/' + sub(basename(c_mergebam.mergebam),'.sorted.b.*$','') + '/BAM_files'
             }
         } # end if blacklist provided
 
         File c_mergebam_afterbklist = select_first([c_merge_rmblklist.intersect_out, c_mergebam.mergebam])
 
+        call samtools.index_bam as c_mergebam_afterbklist_index_m { input: bam=c_mergebam_afterbklist }
```

Comment:
this is redundant. 
ROSE = select_first([c_merge_bklist.indexbam, c_mergeindexstats.indexbam])
BAMtoGFF (.bam) = c_merge_markdup.mkdupbam
BAMtoGFF (.bai) = c_merge_mkdup.indexbam